app/controllers/declaration/SummaryController.scala

```diff
@@ -64,23 +69,41 @@ class SummaryController @Inject()(
   }
 
   val submitDeclaration: Action[AnyContent] = (authenticate andThen verifyEmail andThen journeyType).async { implicit request =>
-    LegalDeclaration.form.bindFromRequest
-      .fold(
+    def submit(form: Form[LegalDeclaration]): Future[Result] =
+      form.bindFromRequest.fold(
         (formWithErrors: Form[LegalDeclaration]) => Future.successful(BadRequest(normalSummaryPage(formWithErrors))),
         legalDeclaration => {
           submissionService.submit(request.eori, request.cacheModel, legalDeclaration).map {
             case Some(submission) =>
               Redirect(routes.ConfirmationController.displayHoldingPage).withSession(session(submission))
-
             case _ => handleError(s"Error from Customs Declarations API")
           }
         }
       )
+
+    lrnIsDuplicate(getLrn(request.cacheModel)).flatMap { result =>
+      result match {
+        case true  => submit(form.copy(errors = Seq(FormError("lrn", "declaration.consignmentReferences.lrn.error.notExpiredYet"))))
+        case false => submit(form)
+      }
+    }
   }
 
   private def containsMandatoryData(declaration: ExportsDeclaration, mode: Mode): Boolean =
     mode.equals(Mode.Draft) || declaration.consignmentReferences.exists(references => references.lrn.nonEmpty)
 
+  private def lrnIsDuplicate(lrn: Option[Lrn])(implicit hc: HeaderCarrier): Future[Boolean] =
+    lrn match {
+      case Some(lrn) => lrnValidator.hasBeenSubmittedInThePast48Hours(lrn)
+      case _         => Future.successful(false)
+    }
+
+  private def getLrn(declaration: ExportsDeclaration): Option[Lrn] =
+    declaration.consignmentReferences.map(_.lrn) match {
+      case Some(lrn) => Some(lrn)
+      case _         => None
+    }
```

Comment:
I removed this method but still had to have a matching statement on `ExportsDeclaration.lrn` because it's an `Option[String]` - hopefully this looks better?